runtime/src/main/java/org/corfudb/runtime/ViewsMaintenance.java

```diff
@@ -23,39 +24,71 @@
 
 @Slf4j
 @NotThreadSafe
-public class ViewsGarbageCollector {
+public class ViewsMaintenance {
 
 
     private long trimMark = Address.NON_ADDRESS;
 
     @Getter
     private boolean started;
 
-    final ScheduledExecutorService gcThread = Executors.newSingleThreadScheduledExecutor(
+    @Getter
+    @Setter
+    long objectSyncDelayInSeconds = 30;
+
+    @Getter
+    @Setter
+    long objectGCPeriodInSeconds = 20 * 60;
```

Comment:
add comments as to what these variables are for.

Reply:
done.